spec/math_spec.rb

```diff
@@ -1101,4 +1101,16 @@
       end
     end
   end
+
+  context "#positive_semidefinite? " do
+    it "should return true for positive_semidefinite? matrix" do
```

Comment:
Don't you also need to test to see if it returns false for a matrix that *isn't* positive semi-definite?